proto/def.bzl

```diff
@@ -146,6 +153,7 @@ go_proto_library = rule(
             providers = [GoLibrary],
             aspects = [_go_proto_aspect],
         ),
+        "go_srcs": attr.label_list(allow_files = go_exts + asm_exts + cgo_exts),
```

Comment:
add this attribute to the call to `attr.label_list()`:

`default = [],`

... and then the `getattr()` above shouldn't be necessary and you can just do `ctx.attr.go_srcs`.